cardano-node/src/Cardano/Tracing/OrphanInstances/Consensus.hs

```diff
@@ -441,6 +449,14 @@ instance ( ConvertRawHash blk
         ChainDB.ScheduledGC slot _difft ->
           "Scheduled a garbage collection for " <> condenseT slot
       ChainDB.TraceOpenEvent ev -> case ev of
+        ChainDB.StartedOpeningDB ->
+          "Started opening db"
+        ChainDB.StartedOpeningImmutableDB ->
+          "Started opening immutable db"
```

Comment:
I noticed that [[3388](https://github.com/input-output-hk/cardano-node/pull/3388](https://github.com/input-output-hk/cardano-node/pull/3388/files#r760611056) says "Started opening Immutable DB" whilst this PR says "Started opening immutable db".

Perhaps the casing should be the same as the other?

Similarly for other strings.